### PR TITLE
Fix invalid names in 'source_pads_surface' generator

### LINKED ISSUE #59: source_pads_surface is broken
The source_pads_surface vertex generator doesn't work. Tested source_pads_external_surface and source_pads_internal_surface and they are both fine, so it's the act of combining them that isn't set up properly.

```
flsimulate : Setup/run of simulation threw exception
[virtual void genvtx::combined_vg::initialize(const ::datatools::properties &, ::datatools::service_manager &, ::genvtx::vg_dict_type &):349: Vertex generator 'source_pads_internal_surface' has no weight information in combined vertex generator 'source_pads_surface' !]

```
Guessing this just missing a weight so it's probably an easy fix if you know where the file is and what weights you want to use. But I don't.

### PR DESCRIPTION
**Pull request by fmauger**
Changes proposed in this pull request:
- Fix invalid names of combined generators in the definition of the ``source_pads_surface`` vertex generator (version 4.1 of the SuperNEMO demonstrator vertex generators setup).

Does this pull request fix any reported issues?
- Fixes #59 .

Additional comments or information
----------------------------------

### CODE REVIEW

File: resources/config/snemo/demonstrator/simulation/vertexes/4.1/generators/source_vg.def [hunk 1]
```diff
@@ -164,8 +164,12 @@ generators : string[2] = "source_pads_internal_surface" "source_pads_external_su
 # relative masses of internal/external strips:
 #
 #@variant_if    geometry:layout/if_basic/source_layout/if_basic|true
-generators.source_pads_external_bulk.absolute_weight : real  =  250.0 # 125 mm * 2
-generators.source_pads_internal_bulk.absolute_weight : real  = 4607.0 # 135.5 mm * 34
+# Date:    2017-10-20
+# Author:  frc
+# Issue:   https://github.com/SuperNEMO-DBD/Falaise/issues/59
+# Comment: Copy/paste error on combined generators' names.
```
Comment (drbenmorgan) [inline resources/config/snemo/demonstrator/simulation/vertexes/4.1/generators/source_vg.def]: Please remove these comments - they're extraneous and reproduce info that's already in the commit log.

File: resources/config/snemo/demonstrator/simulation/vertexes/4.1/generators/source_vg.def [hunk 2]
```diff
@@ -164,8 +164,12 @@ generators : string[2] = "source_pads_internal_surface" "source_pads_external_su
...
+generators.source_pads_external_surface.absolute_weight : real  =  250.0 # 125 mm * 2
+generators.source_pads_internal_surface.absolute_weight : real  = 4607.0 # 135.5 mm * 34
 #@variant_endif geometry:layout/if_basic/source_layout/if_basic
```
Comment (drbenmorgan) [inline resources/config/snemo/demonstrator/simulation/vertexes/4.1/generators/source_vg.def]: Please add a test of running these options in `flsimulate` - see the files and CMake script in source/flsimulate/tests for examples.

**Comment by drbenmorgan** [comment]
Relaunched failed build, was a timeout rather than other error, but check.

**Comment by fmauger** [comment]
Missing CTest entry has been added and validated.
PNG files are left. I don't want to remove them.

**Comment by drbenmorgan** [comment]
@fmauger, did you mean to close this?

**Comment by cherylepatrick** [comment]
Will I get this fix if I brew upgrade?

**Comment by yramachers** [comment]
I just encountered the same issue as Cheryl in #60 , saw that it was raised as an issue and noticed that the small bug is still in the code. Your solution looks fine, just a typo correction but it can't be merged when the PR is closed as I understand it. My only question on the solution would be whether a weighting factor by mass is correct rather than a weighting factor by area for such a surface source.

**Comment by yramachers** [comment]
Ah, scratch that, the unit for the weighting factor is mm, not mass. Got confused by this weighting factor being identical to the one used for the bulk source vg.

**Comment by fmauger** [comment]
Ooops! It was closed by error despite it is ready for merging.

**Pull request merged**